Automattic/jetpack
# Fix domain modal overlaps with Nav-unification in paid-upgrades.php

Added a filter for "admin_body_class" that will append the class "nav-unification-enabled" so that we can identify that nav-unification is enabled. This implementation is a suggestion from D70757-code



Partially fixes [#58572](https://github.com/Automattic/wp-calypso/issues/58572). The scope of this PR is to just add the class to the body and not actually fix the main issue.

#### Changes proposed in this Pull Request:

* The Base admin menu adds a filter upon being constructed.
* The filter handler will add the class "nav-unification-enabled" to "admin_body_class".

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Open your sandboxed simple site.
* Go to https://yoursandboxedsitegoeshere.com/wp-admin/paid-upgrades.php?page=domains
* View the page's source code or inspect the body element.
* You should see that the body tag has the class: "nav-unification-enabled"
* Go to https://yoursandboxedsitegoeshere.com/wp-admin/paid-upgrades.php?page=domains&disable-nav-unification
* View the page's source code or inspect the body element.
* You should see that the body tag does not have the class: "nav-unification-enabled"
